mne/preprocessing/ica.py

```diff
@@ -1202,7 +1339,7 @@ def find_bads_eog(self, inst, ch_name=None, threshold=3.0, start=None,
 
         See Also
         --------
-        find_bads_ecg
+        find_bads_ecg, find_bads_ref
```

Comment:
I think it should be on a the next line to render fine.

Reply:
`numpydoc` does either format correctly

https://11021-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.ICA.html#mne.preprocessing.ICA.find_bads_eog

Reply:
have left as is.